api/src/prescription/campaign/infrastructure/repositories/campaign-profiles-collection-participation-summary-repository.js

```diff
@@ -81,7 +81,7 @@ async function _makeMemoizedGetPlacementProfileForUser(results) {
 
   const sharedResults = results.filter(({ sharedAt }) => sharedAt);
 
-  const sharedResultsChunks = await bluebird.mapSeries(
+  const sharedResultsChunks = await PromiseUtils.mapSeries(
```

Comment:
ici aussi, pourquoi ne pas avoir transformer en boucle `for..of` ?

Reply:
Car le retour du map est utilisé 😉
```js
const sharedResultsChunks = await PromiseUtils.mapSeries(
  chunk(sharedResults, constants.CHUNK_SIZE_CAMPAIGN_RESULT_PROCESSING),
  (sharedResultsChunk) => {
    const userIdsAndDates = sharedResultsChunk.map(({ userId, sharedAt }) => {
      return { userId, sharedAt };
    });

    return placementProfileService.getPlacementProfilesWithSnapshotting({
      userIdsAndDates,
      allowExcessPixAndLevels: false,
      competences,
    });
  },
);

const placementProfiles = sharedResultsChunks.flat();

return (userId) => placementProfiles.find((placementProfile) => placementProfile.userId === userId);
```